diff --git a/MANIFEST.in b/MANIFEST.in
index 9bc2ca6..3287a68 100644
--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,3 +1,3 @@
 include README.rst
 include LICENSE.mdinclude versioneer.py
-include xicam/_version.py
+include xicam/args/_version.py
diff --git a/setup.cfg b/setup.cfg
index 065eed9..9f9e4a8 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -2,6 +2,6 @@
 [versioneer]
 VCS = git
 style = pep440
-versionfile_source = xicam/_version.py
+versionfile_source = xicam/args/_version.py
 versionfile_build =
-tag_prefix =
\ No newline at end of file
+tag_prefix =
diff --git a/setup.py b/setup.py
index a182805..a7dc2a9 100644
--- a/setup.py
+++ b/setup.py
@@ -28,7 +28,7 @@ deps = [
 loosebits = ["numpy.core._methods", "numpy.lib.recfunctions"]
 
 
-from setuptools import setup, find_packages
+from setuptools import setup, find_packages, find_namespace_packages
 
 setup(
     name="xicam",
@@ -66,7 +66,7 @@ setup(
     keywords="synchrotron analysis x-ray scattering tomography ",
     # You can just specify the packages manually here if your project is
     # simple. Or you can use find_packages().
-    packages=find_packages(),
+    packages=find_packages() + find_namespace_packages(include=['xicam.*']),
     package_dir={},
     # Alternatively, if you want to distribute just a my_module.py, uncomment
     # this:
diff --git a/xicam/args.py b/xicam/args/__init__.py
similarity index 100%
rename from xicam/args.py
rename to xicam/args/__init__.py
diff --git a/xicam/_version.py b/xicam/args/_version.py
similarity index 100%
rename from xicam/_version.py
rename to xicam/args/_version.py
diff --git a/xicam/test.py b/xicam/args/test.py
similarity index 100%
rename from xicam/test.py
rename to xicam/args/test.py
